Modules/Microsoft365DSC/DSCResources/MSFT_O365Group/MSFT_O365Group.schema.mof

```diff
@@ -9,8 +9,7 @@ class MSFT_O365Group : OMI_BaseResource
     [Write, Description("Present ensures the group exists, absent ensures it is removed."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] string Ensure;
     [Write, Description("Credentials of the Exchange Global Admin"), EmbeddedInstance("MSFT_Credential")] string Credential;
     [Write, Description("Id of the Azure Active Directory application to authenticate with.")] String ApplicationId;
-    [Write, Description("Id of the Azure Active Directory tenant used for authentication.")] String TenantId;
+    [Write, Description("Name of the Azure Active Directory tenant used for authentication. Format contoso.onmicrosoft.com")] String TenantId;
+    [Write, Description("Secret of the Azure Active Directory tenant used for authentication.")] String ApplicationSecret;
```

Comment:
Shouldn't this be Azure Active Directory application instead of tenant?

Reply:
It's actually the id of the tenant where the app is deployed, so I believe the current phrasing is correct.

Reply:
Shouldn't it be ApplicationId then instead of ApplicationSecret?